com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareCommon.hlsl

```diff
@@ -3,28 +3,26 @@
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Random.hlsl"
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Sampling/Sampling.hlsl"
 
-#if SHADER_API_GLES
 struct AttributesLensFlare
 {
-    float4 positionCS       : POSITION;
-    float2 uv               : TEXCOORD0;
+    uint vertexID : SV_VertexID;
 
 #ifndef FLARE_PREVIEW
     UNITY_VERTEX_INPUT_INSTANCE_ID
 #endif
 };
-#else
+
+#if SHADER_API_GLES
```

Comment:
This change is still wrong, you will have double declaration of AttributeLensFlare and no declaration of Varyings in GLES.
 
 master is correct to me: https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareCommon.hlsl#L6-L25

Reply:
[Fixed]